packages/sdk-bundles-api/src/replay-orchestrator/sdk-to-bundle/execute-replay.ts

```diff
@@ -180,6 +180,11 @@ export interface ReplayExecutionOptions {
   essentialFeaturesOnly: boolean;
 
   vercel?: VercelExecutionSettings;
+
+  /**
+   * If populated, each header will be injected to all requests when fetching resources during a replay.
+   */
+  customRequestHeaders?: Record<string, string>;
```

Comment:
Note: if I can figure out how to cleanly roll the Vercel `deploymentProtectionBypassToken` into `customRequestHeaders` while writing the monorepo PR, I will deprecate the `vercel` field in a subsequent SDK PR.